Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.h

```diff
@@ -276,6 +276,9 @@ public slots:
   /// Enable/disable masking using master volume intensity
   void toggleMasterVolumeIntensityMaskEnabled();
 
+  /// Toggle style of effect buttons to see button labels if needed.
+  void toggleToolButtonStyle();
```

Comment:
For consistency, the method name should include "effect". Also, the style is not a binary state, so it cannot be toggled. Something like toggleEffectButtonLabelVisibility would work.